dotnet/samples/KernelSyntaxExamples/Example00_03_OpenApiSkill_PlayFab.cs

```diff
@@ -20,7 +24,49 @@ public static class Example_00_03_OpenApiSkill_PlayFab
 {
     public static async Task RunAsync()
     {
+        // Simple example
         await SkillImportExample();
+
+        // Example semantic skill for generating PlayFab segments
+        string[] questions = new string[]
+        {
+             "How do I create a segment for Android players in Canada?",
+             "How do I create a segment for players with high risk of churn who have spent over $100?"
+        };
```

Comment:
Can we make the two distinct list of questions into one?
(and then have something like the action-planner picking one of your skills to do the job)